akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -215,7 +215,7 @@ final class MergePreferred[T](val secondaryPorts: Int, val eagerComplete: Boolea
 
     override def preStart(): Unit = {
       tryPull(preferred)
-      shape.inSeq.foreach(tryPull)
+      shape.inArray.foreach(tryPull)
```

Comment:
yes, well, therein lies the rub: if I make it `inlets`, then tests start to fail :-/